src/assets/safe-recovery-module/v0.1.0/social-recovery-module.json

```diff
@@ -0,0 +1,721 @@
+{
+  "released": true,
+  "contractName": "FCLP256Verifier",
```

Comment:
```suggestion
  "contractName": "SocialRecoveryModule",
```

Reply:
Oh wow! Can't believe I missed that. Thanks :)

Reply:
Version was wrong too :see_no_evil: - should be fixed now.